packages/flutter_tools/bin/fuchsia_attach.dart

```diff
@@ -107,7 +113,8 @@ Future<void> main(List<String> args) async {
     overrides: <Type, Generator>{
       FeatureFlags: () => const _FuchsiaFeatureFlags(),
       DeviceManager: () => _FuchsiaDeviceManager(),
-      FuchsiaArtifacts: () => FuchsiaArtifacts(sshConfig: sshConfig, devFinder: devFinder),
+      FuchsiaArtifacts: () => FuchsiaArtifacts(
+        sshConfig: sshConfig, devFinder: devFinder, ffx: ffx),
```

Comment:
nit: formatting (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation)

```suggestion
      FuchsiaArtifacts: () => FuchsiaArtifacts(
        sshConfig: sshConfig,
        devFinder: devFinder,
        ffx: ffx,
      ),
```